torch/distributions/utils.py

```diff
@@ -5,6 +5,10 @@
 from typing import Dict, Any
 
 
+euler_constant = 0.57721566490153286060  # Euler Mascheroni Constant
```

Comment:
Why not read this from torch.distributions.gumbel?

Reply:
I think it's good SWE practice to isolate constants into a separate file?

Reply:
Ah nice I missed that you updated the gumbel file to import this one